units/provisioning_create_host.go

```diff
@@ -156,8 +159,8 @@ func (j *createHostJob) Run(ctx context.Context) {
 			MaxTime: j.host.SpawnOptions.TimeoutSetup.Sub(j.start),
 		})
 	}
-
-	j.AddError(j.createHost(ctx))
+	err = j.createHost(ctx)
```

Comment:
I think the original form was more idiomatic.

Reply:
Sorry, hold over from debugging logging changes I think.